File: Syncfusion_Sample/Windows/Tools.Windows/Samples/2.0/Editors Package/Gradient Label/VB/MainForm.vb
#Region "Copyright Syncfusion Inc. 2001 - 2013"
'
'  Copyright Syncfusion Inc. 2001 - 2013. All rights reserved.
'
'  Use of this code is subject to the terms of our license.
'  A copy of the current license can be obtained at any time by e-mailing
'  licensing@syncfusion.com. Any infringement will be prosecuted under
'  applicable laws. 
'
#End Region


Imports Microsoft.VisualBasic
	Imports System
	Imports System.Drawing
	Imports System.Collections
	Imports System.ComponentModel
	Imports System.Windows.Forms
	Imports System.Data
	Imports Syncfusion.Windows.Forms.Tools
	Imports Syncfusion.Drawing
Namespace GradientLabelDemo

	''' <summary>
	''' Summary description for MainForm.
	''' </summary>
	Public Class MainForm
		Inherits System.Windows.Forms.Form
		Private gradientLabel2 As Syncfusion.Windows.Forms.Tools.GradientLabel
		Private gradientLabel3 As Syncfusion.Windows.Forms.Tools.GradientLabel
		Private gradientLabel4 As Syncfusion.Windows.Forms.Tools.GradientLabel
		Private errorProvider1 As System.Windows.Forms.ErrorProvider
		Private components As IContainer
		Private gradientLabel5 As Syncfusion.Windows.Forms.Tools.GradientLabel
		Private gradientLabel1 As Syncfusion.Windows.Forms.Tools.GradientLabel
		Private gradientPanel1 As Syncfusion.Windows.Forms.Tools.GradientPanel
		Private gradientLabel7 As GradientLabel
		Private gradientLabel6 As GradientLabel

		Private hashLabelNames As Hashtable

		Public Sub New()
			'
			' Required for Windows Form Designer support
			'
			InitializeComponent()

		End Sub

		''' <summary>
		''' Clean up any resources being used.
		''' </summary>
		Protected Overrides Overloads Sub Dispose(ByVal disposing As Boolean)
			If disposing Then
				If Not components Is Nothing Then
					components.Dispose()
				End If
			End If
			MyBase.Dispose(disposing)
		End Sub

		#Region "Windows Form Designer generated code"
		''' <summary>
		''' Required method for Designer support - do not modify
		''' the contents of this method with the code editor.
		''' </summary>
		Private Sub InitializeComponent()
			Me.components = New System.ComponentModel.Container()
			Dim resources As System.ComponentModel.ComponentResourceManager = New System.ComponentModel.ComponentResourceManager(GetType(MainForm))
			Me.gradientLabel2 = New Syncfusion.Windows.Forms.Tools.GradientLabel()
			Me.gradientLabel3 = New Syncfusion.Windows.Forms.Tools.GradientLabel()
			Me.gradientLabel4 = New Syncfusion.Windows.Forms.Tools.GradientLabel()
			Me.errorProvider1 = New System.Windows.Forms.ErrorProvider(Me.components)
			Me.gradientLabel5 = New Syncfusion.Windows.Forms.Tools.GradientLabel()
			Me.gradientLabel1 = New Syncfusion.Windows.Forms.Tools.GradientLabel()
			Me.gradientPanel1 = New Syncfusion.Windows.Forms.Tools.GradientPanel()
			Me.gradientLabel6 = New Syncfusion.Windows.Forms.Tools.GradientLabel()
			Me.gradientLabel7 = New Syncfusion.Windows.Forms.Tools.GradientLabel()
			CType(Me.errorProvider1, System.ComponentModel.ISupportInitialize).BeginInit()
			CType(Me.gradientPanel1, System.ComponentModel.ISupportInitialize).BeginInit()
			Me.gradientPanel1.SuspendLayout()
			Me.SuspendLayout()
			' 
			' gradientLabel2
			' 
			Me.gradientLabel2.BackgroundColor = New Syncfusion.Drawing.BrushInfo(Syncfusion.Drawing.PatternStyle.DiagonalBrick, System.Drawing.SystemColors.Highlight, System.Drawing.Color.FromArgb((CInt(Fix((CByte(255))))), (CInt(Fix((CByte(255))))), (CInt(Fix((CByte(192)))))))
			Me.gradientLabel2.BorderSides = (CType(((((System.Windows.Forms.Border3DSide.Left Or System.Windows.Forms.Border3DSide.Top) Or System.Windows.Forms.Border3DSide.Right) Or System.Windows.Forms.Border3DSide.Bottom) Or System.Windows.Forms.Border3DSide.Middle), System.Windows.Forms.Border3DSide))
			Me.gradientLabel2.BorderStyle = System.Windows.Forms.Border3DStyle.SunkenOuter
			Me.gradientLabel2.Font = New System.Drawing.Font("Arial", 9.75F, System.Drawing.FontStyle.Bold, System.Drawing.GraphicsUnit.Point, (CByte(0)))
			Me.gradientLabel2.ForeColor = System.Drawing.Color.SlateBlue
			Me.gradientLabel2.Location = New System.Drawing.Point(3, 54)
			Me.gradientLabel2.Name = "gradientLabel2"
			Me.gradientLabel2.Size = New System.Drawing.Size(156, 136)
			Me.gradientLabel2.TabIndex = 7
			Me.gradientLabel2.Text = "Background Patterns"
			Me.gradientLabel2.TextAlign = System.Drawing.ContentAlignment.MiddleCenter
			' 
			' gradientLabel3
			' 
			Me.gradientLabel3.BackgroundColor = New Syncfusion.Drawing.BrushInfo(Syncfusion.Drawing.GradientStyle.Vertical, System.Drawing.Color.FromArgb((CInt(Fix((CByte(249))))), (CInt(Fix((CByte(252))))), (CInt(Fix((CByte(248)))))), System.Drawing.Color.FromArgb((CInt(Fix((CByte(212))))), (CInt(Fix((CByte(236))))), (CInt(Fix((CByte(201)))))))
			Me.gradientLabel3.BorderSides = (CType((((System.Windows.Forms.Border3DSide.Left Or System.Windows.Forms.Border3DSide.Top) Or System.Windows.Forms.Border3DSide.Right) Or System.Windows.Forms.Border3DSide.Bottom), System.Windows.Forms.Border3DSide))
			Me.gradientLabel3.BorderStyle = System.Windows.Forms.Border3DStyle.Flat
			Me.gradientLabel3.ForeColor = System.Drawing.SystemColors.ControlText
			Me.gradientLabel3.Location = New System.Drawing.Point(353, 54)
			Me.gradientLabel3.Name = "gradientLabel3"
			Me.gradientLabel3.Size = New System.Drawing.Size(181, 136)
			Me.gradientLabel3.TabIndex = 8
			Me.gradientLabel3.Text = "You have received a new message from Syncfusion."
			Me.gradientLabel3.TextAlign = System.Drawing.ContentAlignment.MiddleCenter
			' 
			' gradientLabel4
			' 
			Me.gradientLabel4.BackgroundColor = New Syncfusion.Drawing.BrushInfo(Syncfusion.Drawing.GradientStyle.PathEllipse, System.Drawing.Color.White, System.Drawing.Color.FromArgb((CInt(Fix((CByte(255))))), (CInt(Fix((CByte(192))))), (CInt(Fix((CByte(128)))))))
			Me.gradientLabel4.BorderSides = (CType((((System.Windows.Forms.Border3DSide.Left Or System.Windows.Forms.Border3DSide.Top) Or System.Windows.Forms.Border3DSide.Right) Or System.Windows.Forms.Border3DSide.Bottom), System.Windows.Forms.Border3DSide))
			Me.gradientLabel4.BorderStyle = System.Windows.Forms.Border3DStyle.Etched
			Me.gradientLabel4.Font = New System.Drawing.Font("Arial", 8.25F, System.Drawing.FontStyle.Regular, System.Drawing.GraphicsUnit.Point, (CByte(0)))
			Me.gradientLabel4.ForeColor = System.Drawing.Color.Teal
			Me.gradientLabel4.Location = New System.Drawing.Point(170, 54)
			Me.gradientLabel4.Name = "gradientLabel4"
			Me.gradientLabel4.Size = New System.Drawing.Size(177, 33)
			Me.gradientLabel4.TabIndex = 9
			Me.gradientLabel4.Text = "10 Feeds Updated"
			Me.gradientLabel4.TextAlign = System.Drawing.ContentAlignment.MiddleCenter
			' 
			' errorProvider1
			' 
			Me.errorProvider1.ContainerControl = Me
			Me.errorProvider1.Icon = (CType(resources.GetObject("errorProvider1.Icon"), System.Drawing.Icon))
			' 
			' gradientLabel5
			' 
			Me.gradientLabel5.BackColor = System.Drawing.Color.FromArgb((CInt(Fix((CByte(255))))), (CInt(Fix((CByte(224))))), (CInt(Fix((CByte(192))))))
			Me.gradientLabel5.BackgroundColor = New Syncfusion.Drawing.BrushInfo(Syncfusion.Drawing.GradientStyle.PathRectangle, System.Drawing.Color.DarkGray, System.Drawing.Color.Black)
			Me.gradientLabel5.BorderSides = System.Windows.Forms.Border3DSide.Left
			Me.gradientLabel5.BorderStyle = System.Windows.Forms.Border3DStyle.Flat
			Me.gradientLabel5.Font = New System.Drawing.Font("Arial", 9.75F, System.Drawing.FontStyle.Bold, System.Drawing.GraphicsUnit.Point, (CByte(0)))
			Me.gradientLabel5.ForeColor = System.Drawing.Color.White
			Me.gradientLabel5.Image = (CType(resources.GetObject("gradientLabel5.Image"), System.Drawing.Image))
			Me.gradientLabel5.ImageAlign = System.Drawing.ContentAlignment.MiddleRight
			Me.gradientLabel5.Location = New System.Drawing.Point(3, 208)
			Me.gradientLabel5.Name = "gradientLabel5"
			Me.gradientLabel5.Size = New System.Drawing.Size(531, 46)
			Me.gradientLabel5.TabIndex = 10
			Me.gradientLabel5.Text = " P O W E R   T O O L S"
			Me.gradientLabel5.TextAlign = System.Drawing.ContentAlignment.MiddleLeft
			' 
			' gradientLabel1
			' 
			Me.gradientLabel1.BackColor = System.Drawing.Color.FromArgb((CInt(Fix((CByte(192))))), (CInt(Fix((CByte(192))))), (CInt(Fix((CByte(255))))))
			Me.gradientLabel1.BackgroundColor = New Syncfusion.Drawing.BrushInfo(Syncfusion.Drawing.GradientStyle.Vertical, New System.Drawing.Color() { System.Drawing.Color.FromArgb((CInt(Fix((CByte(116))))), (CInt(Fix((CByte(117))))), (CInt(Fix((CByte(148)))))), System.Drawing.Color.FromArgb((CInt(Fix((CByte(116))))), (CInt(Fix((CByte(117))))), (CInt(Fix((CByte(148)))))), System.Drawing.Color.FromArgb((CInt(Fix((CByte(116))))), (CInt(Fix((CByte(117))))), (CInt(Fix((CByte(148)))))), System.Drawing.Color.FromArgb((CInt(Fix((CByte(140))))), (CInt(Fix((CByte(140))))), (CInt(Fix((CByte(172)))))), System.Drawing.Color.FromArgb((CInt(Fix((CByte(156))))), (CInt(Fix((CByte(157))))), (CInt(Fix((CByte(180)))))), System.Drawing.Color.FromArgb((CInt(Fix((CByte(164))))), (CInt(Fix((CByte(166))))), (CInt(Fix((CByte(188))))))})
			Me.gradientLabel1.BorderSides = (CType(((((System.Windows.Forms.Border3DSide.Left Or System.Windows.Forms.Border3DSide.Top) Or System.Windows.Forms.Border3DSide.Right) Or System.Windows.Forms.Border3DSide.Bottom) Or System.Windows.Forms.Border3DSide.Middle), System.Windows.Forms.Border3DSide))
			Me.gradientLabel1.BorderStyle = System.Windows.Forms.Border3DStyle.Flat
			Me.gradientLabel1.Dock = System.Windows.Forms.DockStyle.Top
			Me.gradientLabel1.Font = New System.Drawing.Font("Arial", 12F, System.Drawing.FontStyle.Bold, System.Drawing.GraphicsUnit.Point, (CByte(0)))
			Me.gradientLabel1.ForeColor = System.Drawing.Color.White
			Me.gradientLabel1.Location = New System.Drawing.Point(0, 0)
			Me.gradientLabel1.Name = "gradientLabel1"
			Me.gradientLabel1.Size = New System.Drawing.Size(544, 40)
			Me.gradientLabel1.TabIndex = 16
			Me.gradientLabel1.Text = "Column Editor"
			Me.gradientLabel1.TextAlign = System.Drawing.ContentAlignment.MiddleLeft
			' 
			' gradientPanel1
			' 
			Me.gradientPanel1.BorderColor = System.Drawing.Color.FromArgb((CInt(Fix((CByte(146))))), (CInt(Fix((CByte(145))))), (CInt(Fix((CByte(174))))))
			Me.gradientPanel1.BorderStyle = System.Windows.Forms.BorderStyle.FixedSingle
			Me.gradientPanel1.Controls.Add(Me.gradientLabel1)
			Me.gradientPanel1.Controls.Add(Me.gradientLabel2)
			Me.gradientPanel1.Controls.Add(Me.gradientLabel5)
			Me.gradientPanel1.Controls.Add(Me.gradientLabel3)
			Me.gradientPanel1.Controls.Add(Me.gradientLabel7)
			Me.gradientPanel1.Controls.Add(Me.gradientLabel6)
			Me.gradientPanel1.Controls.Add(Me.gradientLabel4)
			Me.gradientPanel1.Location = New System.Drawing.Point(12, 16)
			Me.gradientPanel1.Name = "gradientPanel1"
			Me.gradientPanel1.Size = New System.Drawing.Size(546, 268)
			Me.gradientPanel1.TabIndex = 22
			' 
			' gradientLabel6
			' 
			Me.gradientLabel6.BackgroundColor = New Syncfusion.Drawing.BrushInfo(Syncfusion.Drawing.GradientStyle.Vertical, System.Drawing.Color.White, System.Drawing.Color.FromArgb((CInt(Fix((CByte(192))))), (CInt(Fix((CByte(192))))), (CInt(Fix((CByte(255)))))))
			Me.gradientLabel6.BorderSides = (CType((((System.Windows.Forms.Border3DSide.Left Or System.Windows.Forms.Border3DSide.Top) Or System.Windows.Forms.Border3DSide.Right) Or System.Windows.Forms.Border3DSide.Bottom), System.Windows.Forms.Border3DSide))
			Me.gradientLabel6.BorderStyle = System.Windows.Forms.Border3DStyle.Etched
			Me.gradientLabel6.Font = New System.Drawing.Font("Arial", 8.25F, System.Drawing.FontStyle.Regular, System.Drawing.GraphicsUnit.Point, (CByte(0)))
			Me.gradientLabel6.ForeColor = System.Drawing.Color.MidnightBlue
			Me.gradientLabel6.Image = (CType(resources.GetObject("gradientLabel6.Image"), System.Drawing.Image))
			Me.gradientLabel6.ImageAlign = System.Drawing.ContentAlignment.MiddleLeft
			Me.gradientLabel6.Location = New System.Drawing.Point(170, 111)
			Me.gradientLabel6.Name = "gradientLabel6"
			Me.gradientLabel6.Size = New System.Drawing.Size(177, 32)
			Me.gradientLabel6.TabIndex = 9
			Me.gradientLabel6.Text = "Attachments"
			Me.gradientLabel6.TextAlign = System.Drawing.ContentAlignment.MiddleCenter
			' 
			' gradientLabel7
			' 
			Me.gradientLabel7.BackgroundColor = New Syncfusion.Drawing.BrushInfo(Syncfusion.Drawing.GradientStyle.PathRectangle, System.Drawing.Color.White, System.Drawing.Color.FromArgb((CInt(Fix((CByte(221))))), (CInt(Fix((CByte(224))))), (CInt(Fix((CByte(227)))))))
			Me.gradientLabel7.BorderSides = (CType(((((System.Windows.Forms.Border3DSide.Left Or System.Windows.Forms.Border3DSide.Top) Or System.Windows.Forms.Border3DSide.Right) Or System.Windows.Forms.Border3DSide.Bottom) Or System.Windows.Forms.Border3DSide.Middle), System.Windows.Forms.Border3DSide))
			Me.gradientLabel7.BorderStyle = System.Windows.Forms.Border3DStyle.Flat
			Me.gradientLabel7.Font = New System.Drawing.Font("Arial", 8.25F, System.Drawing.FontStyle.Bold, System.Drawing.GraphicsUnit.Point, (CByte(0)))
			Me.gradientLabel7.ForeColor = System.Drawing.SystemColors.ControlText
			Me.gradientLabel7.Location = New System.Drawing.Point(170, 170)
			Me.gradientLabel7.Name = "gradientLabel7"
			Me.gradientLabel7.Size = New System.Drawing.Size(177, 20)
			Me.gradientLabel7.TabIndex = 9
			Me.gradientLabel7.Text = "Mail Folders"
			Me.gradientLabel7.TextAlign = System.Drawing.ContentAlignment.MiddleCenter
			' 
			' MainForm
			' 
			Me.AutoScaleBaseSize = New System.Drawing.Size(5, 13)
			Me.BackColor = System.Drawing.Color.FromArgb((CInt(Fix((CByte(246))))), (CInt(Fix((CByte(249))))), (CInt(Fix((CByte(254))))))
			Me.ClientSize = New System.Drawing.Size(570, 299)
			Me.Controls.Add(Me.gradientPanel1)
			Me.FormBorderStyle = System.Windows.Forms.FormBorderStyle.FixedDialog
			Me.Icon = (CType(resources.GetObject("$this.Icon"), System.Drawing.Icon))
			Me.MaximizeBox = False
			Me.Name = "MainForm"
			Me.StartPosition = System.Windows.Forms.FormStartPosition.CenterScreen
			Me.Text = "Gradient Label"
			CType(Me.errorProvider1, System.ComponentModel.ISupportInitialize).EndInit()
			CType(Me.gradientPanel1, System.ComponentModel.ISupportInitialize).EndInit()
			Me.gradientPanel1.ResumeLayout(False)
			Me.ResumeLayout(False)

		End Sub
		#End Region

		''' <summary>
		''' The main entry point for the application.
		''' </summary>
		<STAThread> _
		Public Shared Sub Main()
			Application.Run(New MainForm())
		End Sub
	End Class
End Namespace

File: Syncfusion_Sample/Windows/Tools.Windows/Samples/2.0/Editors Package/Gradient Label/CS/MainForm.cs
#region Copyright Syncfusion Inc. 2001 - 2013
//
//  Copyright Syncfusion Inc. 2001 - 2013. All rights reserved.
//
//  Use of this code is subject to the terms of our license.
//  A copy of the current license can be obtained at any time by e-mailing
//  licensing@syncfusion.com. Any infringement will be prosecuted under
//  applicable laws. 
//
#endregion

namespace GradientLabelDemo
{
	using System;
	using System.Drawing;
	using System.Collections;
	using System.ComponentModel;
	using System.Windows.Forms;
	using System.Data;
	using Syncfusion.Windows.Forms.Tools;
	using Syncfusion.Drawing;
	
	/// <summary>
	/// Summary description for MainForm.
	/// </summary>
    public class MainForm : System.Windows.Forms.Form
    {
        private Syncfusion.Windows.Forms.Tools.GradientLabel gradientLabel2;
        private Syncfusion.Windows.Forms.Tools.GradientLabel gradientLabel3;
        private Syncfusion.Windows.Forms.Tools.GradientLabel gradientLabel4;
        private System.Windows.Forms.ErrorProvider errorProvider1;
        private IContainer components;
        private Syncfusion.Windows.Forms.Tools.GradientLabel gradientLabel5;
        private Syncfusion.Windows.Forms.Tools.GradientLabel gradientLabel1;
        private Syncfusion.Windows.Forms.Tools.GradientPanel gradientPanel1;
        private GradientLabel gradientLabel7;
        private GradientLabel gradientLabel6;        

        public MainForm()
        {
            //
            // Required for Windows Form Designer support
            //
            InitializeComponent();

        }

        /// <summary>
        /// Clean up any resources being used.
        /// </summary>
        protected override void Dispose(bool disposing)
        {
            if (disposing)
            {
                if (components != null)
                {
                    components.Dispose();
                }
            }
            base.Dispose(disposing);
        }

        #region Windows Form Designer generated code
        /// <summary>
        /// Required method for Designer support - do not modify
        /// the contents of this method with the code editor.
        /// </summary>
        private void InitializeComponent()
        {
            this.components = new System.ComponentModel.Container();
            System.ComponentModel.ComponentResourceManager resources = new System.ComponentModel.ComponentResourceManager(typeof(MainForm));
            this.gradientLabel2 = new Syncfusion.Windows.Forms.Tools.GradientLabel();
            this.gradientLabel3 = new Syncfusion.Windows.Forms.Tools.GradientLabel();
            this.gradientLabel4 = new Syncfusion.Windows.Forms.Tools.GradientLabel();
            this.errorProvider1 = new System.Windows.Forms.ErrorProvider(this.components);
            this.gradientLabel5 = new Syncfusion.Windows.Forms.Tools.GradientLabel();
            this.gradientLabel1 = new Syncfusion.Windows.Forms.Tools.GradientLabel();
            this.gradientPanel1 = new Syncfusion.Windows.Forms.Tools.GradientPanel();
            this.gradientLabel6 = new Syncfusion.Windows.Forms.Tools.GradientLabel();
            this.gradientLabel7 = new Syncfusion.Windows.Forms.Tools.GradientLabel();
            ((System.ComponentModel.ISupportInitialize)(this.errorProvider1)).BeginInit();
            ((System.ComponentModel.ISupportInitialize)(this.gradientPanel1)).BeginInit();
            this.gradientPanel1.SuspendLayout();
            this.SuspendLayout();
            // 
            // gradientLabel2
            // 
            this.gradientLabel2.BackgroundColor = new Syncfusion.Drawing.BrushInfo(Syncfusion.Drawing.PatternStyle.DiagonalBrick, System.Drawing.SystemColors.Highlight, System.Drawing.Color.FromArgb(((int)(((byte)(255)))), ((int)(((byte)(255)))), ((int)(((byte)(192))))));
            this.gradientLabel2.BorderSides = ((System.Windows.Forms.Border3DSide)(((((System.Windows.Forms.Border3DSide.Left | System.Windows.Forms.Border3DSide.Top)
                        | System.Windows.Forms.Border3DSide.Right)
                        | System.Windows.Forms.Border3DSide.Bottom)
                        | System.Windows.Forms.Border3DSide.Middle)));
            this.gradientLabel2.BorderStyle = System.Windows.Forms.Border3DStyle.SunkenOuter;
            this.gradientLabel2.Font = new System.Drawing.Font("Arial", 9.75F, System.Drawing.FontStyle.Bold, System.Drawing.GraphicsUnit.Point, ((byte)(0)));
            this.gradientLabel2.ForeColor = System.Drawing.Color.SlateBlue;
            this.gradientLabel2.Location = new System.Drawing.Point(3, 54);
            this.gradientLabel2.Name = "gradientLabel2";
            this.gradientLabel2.Size = new System.Drawing.Size(156, 136);
            this.gradientLabel2.TabIndex = 7;
            this.gradientLabel2.Text = "Background Patterns";
            this.gradientLabel2.TextAlign = System.Drawing.ContentAlignment.MiddleCenter;
            // 
            // gradientLabel3
            // 
            this.gradientLabel3.BackgroundColor = new Syncfusion.Drawing.BrushInfo(Syncfusion.Drawing.GradientStyle.Vertical, System.Drawing.Color.FromArgb(((int)(((byte)(249)))), ((int)(((byte)(252)))), ((int)(((byte)(248))))), System.Drawing.Color.FromArgb(((int)(((byte)(212)))), ((int)(((byte)(236)))), ((int)(((byte)(201))))));
            this.gradientLabel3.BorderSides = ((System.Windows.Forms.Border3DSide)((((System.Windows.Forms.Border3DSide.Left | System.Windows.Forms.Border3DSide.Top)
                        | System.Windows.Forms.Border3DSide.Right)
                        | System.Windows.Forms.Border3DSide.Bottom)));
            this.gradientLabel3.BorderStyle = System.Windows.Forms.Border3DStyle.Flat;
            this.gradientLabel3.ForeColor = System.Drawing.SystemColors.ControlText;
            this.gradientLabel3.Location = new System.Drawing.Point(353, 54);
            this.gradientLabel3.Name = "gradientLabel3";
            this.gradientLabel3.Size = new System.Drawing.Size(181, 136);
            this.gradientLabel3.TabIndex = 8;
            this.gradientLabel3.Text = "You have received a new message from Syncfusion.";
            this.gradientLabel3.TextAlign = System.Drawing.ContentAlignment.MiddleCenter;
            // 
            // gradientLabel4
            // 
            this.gradientLabel4.BackgroundColor = new Syncfusion.Drawing.BrushInfo(Syncfusion.Drawing.GradientStyle.PathEllipse, System.Drawing.Color.White, System.Drawing.Color.FromArgb(((int)(((byte)(255)))), ((int)(((byte)(192)))), ((int)(((byte)(128))))));
            this.gradientLabel4.BorderSides = ((System.Windows.Forms.Border3DSide)((((System.Windows.Forms.Border3DSide.Left | System.Windows.Forms.Border3DSide.Top)
                        | System.Windows.Forms.Border3DSide.Right)
                        | System.Windows.Forms.Border3DSide.Bottom)));
            this.gradientLabel4.BorderStyle = System.Windows.Forms.Border3DStyle.Etched;
            this.gradientLabel4.Font = new System.Drawing.Font("Arial", 8.25F, System.Drawing.FontStyle.Regular, System.Drawing.GraphicsUnit.Point, ((byte)(0)));
            this.gradientLabel4.ForeColor = System.Drawing.Color.Teal;
            this.gradientLabel4.Location = new System.Drawing.Point(170, 54);
            this.gradientLabel4.Name = "gradientLabel4";
            this.gradientLabel4.Size = new System.Drawing.Size(177, 33);
            this.gradientLabel4.TabIndex = 9;
            this.gradientLabel4.Text = "10 Feeds Updated";
            this.gradientLabel4.TextAlign = System.Drawing.ContentAlignment.MiddleCenter;
            // 
            // errorProvider1
            // 
            this.errorProvider1.ContainerControl = this;
            this.errorProvider1.Icon = ((System.Drawing.Icon)(resources.GetObject("errorProvider1.Icon")));
            // 
            // gradientLabel5
            // 
            this.gradientLabel5.BackColor = System.Drawing.Color.FromArgb(((int)(((byte)(255)))), ((int)(((byte)(224)))), ((int)(((byte)(192)))));
            this.gradientLabel5.BackgroundColor = new Syncfusion.Drawing.BrushInfo(Syncfusion.Drawing.GradientStyle.PathRectangle, System.Drawing.Color.DarkGray, System.Drawing.Color.Black);
            this.gradientLabel5.BorderSides = System.Windows.Forms.Border3DSide.Left;
            this.gradientLabel5.BorderStyle = System.Windows.Forms.Border3DStyle.Flat;
            this.gradientLabel5.Font = new System.Drawing.Font("Arial", 9.75F, System.Drawing.FontStyle.Bold, System.Drawing.GraphicsUnit.Point, ((byte)(0)));
            this.gradientLabel5.ForeColor = System.Drawing.Color.White;
            this.gradientLabel5.Image = ((System.Drawing.Image)(resources.GetObject("gradientLabel5.Image")));
            this.gradientLabel5.ImageAlign = System.Drawing.ContentAlignment.MiddleRight;
            this.gradientLabel5.Location = new System.Drawing.Point(3, 208);
            this.gradientLabel5.Name = "gradientLabel5";
            this.gradientLabel5.Size = new System.Drawing.Size(531, 46);
            this.gradientLabel5.TabIndex = 10;
            this.gradientLabel5.Text = " P O W E R   T O O L S";
            this.gradientLabel5.TextAlign = System.Drawing.ContentAlignment.MiddleLeft;
            // 
            // gradientLabel1
            // 
            this.gradientLabel1.BackColor = System.Drawing.Color.FromArgb(((int)(((byte)(192)))), ((int)(((byte)(192)))), ((int)(((byte)(255)))));
            this.gradientLabel1.BackgroundColor = new Syncfusion.Drawing.BrushInfo(Syncfusion.Drawing.GradientStyle.Vertical, new System.Drawing.Color[] {
            System.Drawing.Color.FromArgb(((int)(((byte)(116)))), ((int)(((byte)(117)))), ((int)(((byte)(148))))),
            System.Drawing.Color.FromArgb(((int)(((byte)(116)))), ((int)(((byte)(117)))), ((int)(((byte)(148))))),
            System.Drawing.Color.FromArgb(((int)(((byte)(116)))), ((int)(((byte)(117)))), ((int)(((byte)(148))))),
            System.Drawing.Color.FromArgb(((int)(((byte)(140)))), ((int)(((byte)(140)))), ((int)(((byte)(172))))),
            System.Drawing.Color.FromArgb(((int)(((byte)(156)))), ((int)(((byte)(157)))), ((int)(((byte)(180))))),
            System.Drawing.Color.FromArgb(((int)(((byte)(164)))), ((int)(((byte)(166)))), ((int)(((byte)(188)))))});
            this.gradientLabel1.BorderSides = ((System.Windows.Forms.Border3DSide)(((((System.Windows.Forms.Border3DSide.Left | System.Windows.Forms.Border3DSide.Top)
                        | System.Windows.Forms.Border3DSide.Right)
                        | System.Windows.Forms.Border3DSide.Bottom)
                        | System.Windows.Forms.Border3DSide.Middle)));
            this.gradientLabel1.BorderStyle = System.Windows.Forms.Border3DStyle.Flat;
            this.gradientLabel1.Dock = System.Windows.Forms.DockStyle.Top;
            this.gradientLabel1.Font = new System.Drawing.Font("Arial", 12F, System.Drawing.FontStyle.Bold, System.Drawing.GraphicsUnit.Point, ((byte)(0)));
            this.gradientLabel1.ForeColor = System.Drawing.Color.White;
            this.gradientLabel1.Location = new System.Drawing.Point(0, 0);
            this.gradientLabel1.Name = "gradientLabel1";
            this.gradientLabel1.Size = new System.Drawing.Size(544, 40);
            this.gradientLabel1.TabIndex = 16;
            this.gradientLabel1.Text = "Column Editor";
            this.gradientLabel1.TextAlign = System.Drawing.ContentAlignment.MiddleLeft;
            // 
            // gradientPanel1
            // 
            this.gradientPanel1.BorderColor = System.Drawing.Color.FromArgb(((int)(((byte)(146)))), ((int)(((byte)(145)))), ((int)(((byte)(174)))));
            this.gradientPanel1.BorderStyle = System.Windows.Forms.BorderStyle.FixedSingle;
            this.gradientPanel1.Controls.Add(this.gradientLabel1);
            this.gradientPanel1.Controls.Add(this.gradientLabel2);
            this.gradientPanel1.Controls.Add(this.gradientLabel5);
            this.gradientPanel1.Controls.Add(this.gradientLabel3);
            this.gradientPanel1.Controls.Add(this.gradientLabel7);
            this.gradientPanel1.Controls.Add(this.gradientLabel6);
            this.gradientPanel1.Controls.Add(this.gradientLabel4);
            this.gradientPanel1.Location = new System.Drawing.Point(12, 16);
            this.gradientPanel1.Name = "gradientPanel1";
            this.gradientPanel1.Size = new System.Drawing.Size(546, 268);
            this.gradientPanel1.TabIndex = 22;
            // 
            // gradientLabel6
            // 
            this.gradientLabel6.BackgroundColor = new Syncfusion.Drawing.BrushInfo(Syncfusion.Drawing.GradientStyle.Vertical, System.Drawing.Color.White, System.Drawing.Color.FromArgb(((int)(((byte)(192)))), ((int)(((byte)(192)))), ((int)(((byte)(255))))));
            this.gradientLabel6.BorderSides = ((System.Windows.Forms.Border3DSide)((((System.Windows.Forms.Border3DSide.Left | System.Windows.Forms.Border3DSide.Top)
                        | System.Windows.Forms.Border3DSide.Right)
                        | System.Windows.Forms.Border3DSide.Bottom)));
            this.gradientLabel6.BorderStyle = System.Windows.Forms.Border3DStyle.Etched;
            this.gradientLabel6.Font = new System.Drawing.Font("Arial", 8.25F, System.Drawing.FontStyle.Regular, System.Drawing.GraphicsUnit.Point, ((byte)(0)));
            this.gradientLabel6.ForeColor = System.Drawing.Color.MidnightBlue;
            this.gradientLabel6.Image = ((System.Drawing.Image)(resources.GetObject("gradientLabel6.Image")));
            this.gradientLabel6.ImageAlign = System.Drawing.ContentAlignment.MiddleLeft;
            this.gradientLabel6.Location = new System.Drawing.Point(170, 111);
            this.gradientLabel6.Name = "gradientLabel6";
            this.gradientLabel6.Size = new System.Drawing.Size(177, 32);
            this.gradientLabel6.TabIndex = 9;
            this.gradientLabel6.Text = "Attachments";
            this.gradientLabel6.TextAlign = System.Drawing.ContentAlignment.MiddleCenter;
            // 
            // gradientLabel7
            // 
            this.gradientLabel7.BackgroundColor = new Syncfusion.Drawing.BrushInfo(Syncfusion.Drawing.GradientStyle.PathRectangle, System.Drawing.Color.White, System.Drawing.Color.FromArgb(((int)(((byte)(221)))), ((int)(((byte)(224)))), ((int)(((byte)(227))))));
            this.gradientLabel7.BorderSides = ((System.Windows.Forms.Border3DSide)(((((System.Windows.Forms.Border3DSide.Left | System.Windows.Forms.Border3DSide.Top)
                        | System.Windows.Forms.Border3DSide.Right)
                        | System.Windows.Forms.Border3DSide.Bottom)
                        | System.Windows.Forms.Border3DSide.Middle)));
            this.gradientLabel7.BorderStyle = System.Windows.Forms.Border3DStyle.Flat;
            this.gradientLabel7.Font = new System.Drawing.Font("Arial", 8.25F, System.Drawing.FontStyle.Bold, System.Drawing.GraphicsUnit.Point, ((byte)(0)));
            this.gradientLabel7.ForeColor = System.Drawing.SystemColors.ControlText;
            this.gradientLabel7.Location = new System.Drawing.Point(170, 170);
            this.gradientLabel7.Name = "gradientLabel7";
            this.gradientLabel7.Size = new System.Drawing.Size(177, 20);
            this.gradientLabel7.TabIndex = 9;
            this.gradientLabel7.Text = "Mail Folders";
            this.gradientLabel7.TextAlign = System.Drawing.ContentAlignment.MiddleCenter;
            // 
            // MainForm
            // 
            this.AutoScaleBaseSize = new System.Drawing.Size(5, 13);
            this.BackColor = System.Drawing.Color.FromArgb(((int)(((byte)(246)))), ((int)(((byte)(249)))), ((int)(((byte)(254)))));
            this.ClientSize = new System.Drawing.Size(570, 299);
            this.Controls.Add(this.gradientPanel1);
            this.FormBorderStyle = System.Windows.Forms.FormBorderStyle.FixedDialog;
            this.Icon = ((System.Drawing.Icon)(resources.GetObject("$this.Icon")));
            this.MaximizeBox = false;
            this.Name = "MainForm";
            this.StartPosition = System.Windows.Forms.FormStartPosition.CenterScreen;
            this.Text = "Gradient Label ";
            ((System.ComponentModel.ISupportInitialize)(this.errorProvider1)).EndInit();
            ((System.ComponentModel.ISupportInitialize)(this.gradientPanel1)).EndInit();
            this.gradientPanel1.ResumeLayout(false);
            this.ResumeLayout(false);

        }
        #endregion

        /// <summary>
        /// The main entry point for the application.
        /// </summary>
        [STAThread]
        public static void Main()
        {
            Application.Run(new MainForm());
        }
    }
}
